optifit backend/pose_server.py

```
@@ -0,0 +1,315 @@
+            # Calculate angles
+            left_knee_angle = self.calculate_angle(left_hip, left_knee, left_ankle)
+            right_knee_angle = self.calculate_angle(right_hip, right_knee, right_ankle)
+            left_hip_angle = self.calculate_angle(left_shoulder, left_hip, left_knee)
+            right_hip_angle = self.calculate_angle(right_shoulder, right_hip, right_knee)
+            
+            # Average angles for bilateral assessment
+            knee_angle = (left_knee_angle + right_knee_angle) / 2
+            hip_angle = (left_hip_angle + right_hip_angle) / 2
+            
+            # Calculate torso lean
+            hip_center_y = (left_hip[1] + right_hip[1]) / 2
```

Ruff (0.14.2)

Local variable `hip_center_y` is assigned to but never used

Remove assignment to unused variable `hip_center_y`

(F841)